server/handle_server_create_account.go

```diff
@@ -160,22 +163,29 @@ func (s *Server) handleCreateAccount(e echo.Context) error {
 		SigningKey:            k.Bytes(),
 	}
 
-	actor := models.Actor{
-		Did:    signupDid,
-		Handle: request.Handle,
-	}
-
-	if err := s.db.Create(&urepo, nil).Error; err != nil {
-		s.logger.Error("error inserting new repo", "error", err)
-		return helpers.ServerError(e, nil)
-	}
+	if actor == nil {
+		actor = &models.Actor{
+			Did:    signupDid,
+			Handle: request.Handle,
+		}
 
-	if err := s.db.Create(&actor, nil).Error; err != nil {
-		s.logger.Error("error inserting new actor", "error", err)
-		return helpers.ServerError(e, nil)
+		if err := s.db.Create(&urepo, nil).Error; err != nil {
```

Comment:
These creates are the same, just moved into a "if the actor doesn't already exist"